.github/workflows/build.yml

```diff
@@ -39,28 +39,25 @@ jobs:
       matrix:
         php: [ 8.1, 8.2, 8.3, 8.4 ]
         composer-flags: [ "" ]
-        symfony-version: [ "" ]
         include:
           - php: 8.1
-            symfony-version: '^5.4'
+            composer-flags: "--with=symfony/yaml:^5.4"
           - php: 8.1
-            symfony-version: '^6.4'
+            composer-flags: "--with=symfony/yaml:^6.4"
     steps:
       - uses: actions/checkout@v4
-
```

Comment:
@uuf6429 and I have different opinions on whether the actions yaml is more readable with (my preference) or without (his) a blank line between each step as [previously discussed here](https://github.com/Behat/Gherkin/pull/297#discussion_r2051428957). I'm happy to go with what other maintainers decide.

Reply:
I think that having blank lines between steps in a job is much more readable

Reply:
OK, I'll go for it and update the others as well.